runtime/doc/pi_netrw.txt

```diff
@@ -2533,7 +2573,7 @@ your browsing preferences.  (see also: |netrw-settings|)
   *g:netrw_browsex_viewer*	specify user's preference for a viewer: >
 					"kfmclient exec"
 					"gnome-open"
-<
+
```

Comment:
It seems `<` is needed.

Reply:
True, accidentally removed as help files conceal by default these